tests/conftest.py

```diff
@@ -92,8 +92,23 @@ def repository_paths(tmp_path_factory):
     create_repositories(base_path)
 
     repositories = {
-        "high_entropy": base_path / "high_entropy",
-        "low_entropy": base_path / "low_entropy",
+        "test_repo_1": base_path / "test_repo_1",
+        "test_repo_2": base_path / "test_repo_2",
```

Comment:
Consider keeping some sort of adjective or descriptive word with these test repo names to help describe how they're different at a glance. Maybe something about multi- vs single file?